docs/components/concepts/access-control/user-groups.md

```diff
@@ -0,0 +1,28 @@
+---
+id: user-groups
+title: "User groups"
+sidebar_label: "User groups"
+description: "Control the level of access a user or group has to perform tasks in the system via user task access restrictions."
+---
+
+User groups are a way to organize users in the system. They can be used to control the level of access a user or group has to perform tasks in the system via [user task restrictions](user-task-access-restrictions.md).
+
+### Creating a group
+
+To create a group, navigate to the **Organization** section of Console and click on the **Groups** tab.
+
+![Groups Management](../assets/access-control/group-management.png)
+
+Click **Create a group** and fill in the **Name** field with your group name.
+
+![Create a group](../assets/access-control/create-group.png)
+
+#### Adding users to a group
+
+To add users to a group, navigate to the **Organization** section of Console and click on the **Users** tab.
+
+![Groups Members](../assets/access-control/group-members.png)
+
+Select the user you want to add to a group and click **Assign**. Select the group you want to add the user to and click **Assign members**.
```

Comment:
Are both of these actions required? I just clicked on the group and **Assign members**.

Reply:
I just fixed, :) The part of assign members should be in another paragraph.